refactor(curriculum): restructure sudoku solver python project (#54303)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/6606a219f9efbf38ad496f67.md

---
id: 6606a219f9efbf38ad496f67
title: Step 12
challengeType: 20
dashedName: step-12
---

# --description--

> You can use the dot notation to access an instance attribute, too.

Reviewer: ```suggestion
You can also use dot notation to access an instance attribute.
```
